compose.yaml

```diff
@@ -41,6 +42,10 @@ services:
             - driver: "nvidia"
               capabilities: [ gpu ]
               count: "all"
+    volumes:
+      - models:/app/models
+      - venv:/app/venv
```

Comment:
It could make sense to specify the `/app/venv` and `/app/models` directories as `VOLUME` in the `Dockerfile`:

- https://docs.docker.com/reference/dockerfile/#volume

This way, **(a)** they are initialized with the data already present in the base image, and **(b)** users (presumably) don't need to explicitly configure them in their `compose.yaml` file, which can simplify things and **(c)** avoids reinstalling the dependencies after every configuration change or update if the volume was not specified in the `compose.yaml` file for some reason.

Reply:
It already is. Isn't it?

Reply:
Interesting, you are right. It did nothing, so I've removed it.